docs/pages/blog/introducing-mui-base.md

```diff
@@ -0,0 +1,116 @@
+MUI Base, on the other hand, prioritizes customizability over the speed of development.
+This makes it better suited for public-facing projects where pixel-perfect implementation is crucial.
+
+So, why not use any of the headless libraries already present on the market instead?
+There are other alternatives like Headless UI, Radix UI, and React Aria, to name a few.
+MUI Base's significant advantage is that it takes the best parts from Material UI, which is a complete, mature library.
+It is free from many of the issues that have been reported and fixed in Material UI.
+Also, many of the improvements to Material UI proposed by the community over the years are included in MUI Base.
+
+While they were created for different use cases, these two libraries share many common features and design patterns, and are intended to be complementary to one another within the MUI ecosystem.
+MUI Base's API will be familiar to you if you've used Material UI before, making it easy to migrate a project—or just a part of it—from one library to the other if needed.
+We aim to create [unstyled versions of all Material UI components](https://github.com/mui/material-ui/issues/27170) (that make sense to do so).
+
+## Components and hooks
+
+MUI Base offers two kinds of building blocks: unstyled components and hooks.
+
+Components are more straightforward to use of the two.
+Place a component on a page, add your own styles, and it's ready to go!
+It's important to note that you are not limited to the styling options available in Material UI.
+You can, of course, still use [MUI System](https://mui.com/system/](https://mui.com/system/getting-started/overview/), but if you prefer Emotion, Tailwind CSS, plain CSS, or any other styling engine, they are available too!
+Check out the [Working with Tailwind CSS guide](/base/guides/working-with-tailwind-css/) for an example of using this library.
+
+In contrast to Material UI, MUI Base's components do not have any default styles.
+They provide functionality and structure, while designers and developers are responsible for the visuals.
+
+Each unstyled component lets you modify or override its _slots_—smaller subcomponents representing the interior elements that comprise the component's DOM structure.
+For example, a `SwitchUnstyled` contains the root, thumb, input, and track slots.
+You can control props passed to each of these slots (including `className`) based on the component's state, and even replace the default slot components with your own.
+
+<img src="/static/blog/introducing-mui-base/switch-slots.png" style="width: 692px; aspect-ratio: 173/80; margin-bottom: 24px;" loading="lazy" alt="Depiction of SwitchUnstyled components' slots" />
+
+See how it works on the live demo:
+
+<iframe src="https://codesandbox.io/embed/mui-base-switch-overview-frsm5f?fontsize=12&hidenavigation=1&module=%2Fsrc%2FMySwitch.tsx&theme=dark"
+  style="width:100%; height:500px; border:0; border-radius: 10px; overflow:hidden; margin-bottom: 24px"
```

Comment:
Would it make sense to have something a bit smaller, like?

```suggestion
  style="width:100%; height:350px; border:0; border-radius: 10px; overflow:hidden; margin-bottom: 24px"
```

On my screen, it takes most of the view height. 

---

As a side note, I have pinned the dependencies in the sandbox. It's a blog post, I don't think that we will want to come back in the future to handle breaking changes. 

<img width="349" alt="Screenshot 2022-09-05 at 17 00 39" src="https://user-images.githubusercontent.com/3165635/188477473-7e81ff36-17c1-41bf-9ef0-ced30d2b028f.png">

I have also frozen it to avoid unintended changes

<img width="275" alt="Screenshot 2022-09-05 at 17 02 10" src="https://user-images.githubusercontent.com/3165635/188477537-4a24835d-c5db-4407-994a-36533973ab79.png">

Reply:
Sure, can be.